apps/ledger-live-desktop/src/renderer/screens/settings/sections/General/index.tsx

```diff
@@ -11,13 +11,25 @@ import PasswordButton from "./PasswordButton";
 import PasswordAutoLockSelect from "./PasswordAutoLockSelect";
 import SentryLogsButton from "./SentryLogsButton";
 import ShareAnalyticsButton from "./ShareAnalyticsButton";
+import SharePersonnalRecoButtonFF from "./SharePersonalRecoButtonFF";
+import ShareAnalyticsButtonFF from "./ShareAnalyticsButtonFF";
 import { hasPasswordSelector } from "~/renderer/reducers/application";
 import { useInitSupportedCounterValues } from "~/renderer/hooks/useInitSupportedCounterValues";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
 
 const SectionGeneral = () => {
   const hasPassword = useSelector(hasPasswordSelector);
   const { t } = useTranslation();
   useInitSupportedCounterValues();
+  const lldAnalyticsOptInPromptFlag = useFeature("lldAnalyticsOptInPrompt");
+  const [isFeatureFlagsAnalyticsEnabled, setIsFeatureFlagsAnalyticsEnabled] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    if (lldAnalyticsOptInPromptFlag?.enabled) {
+      setIsFeatureFlagsAnalyticsEnabled(true);
+    }
+  }, [lldAnalyticsOptInPromptFlag]);
```

Comment:
You just need that line and then do
```suggestion
  const lldAnalyticsOptInPromptFlag = useFeature("lldAnalyticsOptInPrompt");

{lldAnalyticsOptInPromptFlag?.enabled && (<YourComponent>}


```